pkg/cmd/pipeline/logs.go

```diff
@@ -53,15 +53,25 @@ func logCommand(p cli.Params) *cobra.Command {
 	}
 
 	eg := `
-  # show the logs of the latest pipelinerun of given pipeline
-	tkn pipeline logs foo -n bar
+  # show logs interactively for no inputs
+    tkn pipeline logs -n namespace
 
+  # show logs interactively for given pipeline
```

Comment:
same as above:  "interactive mode: shows log of the pipeline-run of the given pipeline"